ProdDesProject/Group8
# Health bar

### Health Bar for Duudmayne

Created a simple health bar for dudeman inside the Sample scene. Each time you press 'F' the player takes damage. Player has a max health of 5, and once the health reaches 0, the game restarts after a few seconds.

**Testing**

- Check out the health bar inside the sample scene

- Check that the health bar works as intended (When you start the game, the bar should be full and green. When you take 2 damage (current health 3) the bar turns yellow. When you take 2 more damage (current health 1) the bar turns red.

- After final point of damage (current health 0), you should not be able to move/jump or do anything with the character, and the game should restart automatically after a few seconds.

**Insane death animation for Dudeman to be added later**
